ticdc/ticdc-canal-json.md

```diff
@@ -258,6 +258,45 @@ The following table shows the mapping relationships between Java SQL Types in Ti
 
 For more information about Java SQL Types, see [Java SQL Class Types](https://docs.oracle.com/javase/8/docs/api/java/sql/Types.html).
 
+## Binary and Blob types
+
+TiCDC encodes [binary types](/data-type-string.md#binary-type) in the Canal-JSON format by converting each byte to its character representation as follows:
+
+- Printable characters are represented using the ISO/IEC 8859-1 character encodings. 
+- Non-printable characters and certain characters with special meaning in HTML are represented using their UTF-8 escape sequence. 
+
+The following table shows the detailed representation information.
+
+| Character type                | Value range | Character representation |
+| :---------------------------| :-----------| :---------------------|
+| Control characters          | [0, 31]     | UTF-8 escape (such as `\u0000` through `\u001F`) |
+| Horizontal tab              | [9]         | `\t`                    |
+| Line feed                   | [10]        | `\n`                    |
+| Cariage return              | [13]        | `\r`                    |
```

Comment:
```suggestion
| Carriage return              | [13]        | `\r`                    |
```

Reply:
Addressed in [90f3244](https://github.com/pingcap/docs/pull/13832/commits/90f324453ed9a0f8973bd08b2afae26a35852857)